src/components/discover/DiscoverSearchInput.js

```diff
@@ -26,6 +25,7 @@ import styled from '@/styled-thing';
 import { margin, padding } from '@/styles';
 import { deviceUtils, ethereumUtils } from '@/utils';
 import networkInfo from '@/helpers/networkInfo';
+import DiscoverSheetContext from '@/screens/discover/context/DiscoverScreenContext';
```

Comment:
Nit: don't change this now, doesn't really matter, but I don't think we need this level of granularity in our sub-directories 😛 this component could just live at `/screens/discover/DiscoverScreenContext.tsx` imo. Appreciate the effort here!

Reply:
Makes sense, definitely it could be just one container unless it is a huge component, thanks for feedback!